README.md

```diff
@@ -1,2 +1,164 @@
-# declarative-custom-elements
-Explainer draft for specifying an approach to Declarative Custom Elements
+In reference to https://github.com/WICG/aom/issues/192
+
+```html
+<fancy-label
+    for="input"
+    id="label"
+>
+  <!-- I've not seen talk of offering "for" functionality on other elements, so we likely need to leverage labelledby instead -->
+  #shadow-root
+    <label>Label</label>
+</fancy-label>
+<fancy-input
+    id="input"
+    aria-labelledby="label"
+    role="combobox"
+    aria-controls="listbox"
+    aria-expanded="false"
+    aria-activedescendant="listbox"
+>
+  #shadow-root delegates='aria-labelledby role aria-controls aria-expanded aria-activedescendant'
+    <!-- https://leobalter.github.io/cross-root-aria-delegation/ doesn't currently include 'aria-labelledby', but should? -->
+    <!-- https://leobalter.github.io/cross-root-aria-delegation/ doesn't currently include 'role', but should? -->
+    <!-- https://leobalter.github.io/cross-root-aria-delegation/ doesn't currently include 'aria-controls', but should? -->
+    <!-- https://leobalter.github.io/cross-root-aria-delegation/ doesn't currently include 'aria-activedescendant', but should? -->
+    <input type="text" autoAriaLabelledby autoRole autoAriaExpanded autoActivedescendant>
+</fancy-input>
+<fancy-listbox
+    id="listbox"
+>
+  #shadow-root
+    <ul role="listbox">
+      <!-- OP skipped over the inability to use this DOM structure, but no need to nitpick -->
+      <!-- Placing the role here implies that we can reflect the element out of the shadow DOM in some way,
```

Comment:
This `role` _might_ ONLY manage down in a way that it wouldn't need to be reflected. Review with browsers.